packages/playwright/src/common/test.ts

```diff
@@ -54,9 +54,11 @@ export class Suite extends Base implements SuitePrivate {
   _fullProject: FullProjectInternal | undefined;
   _fileId: string | undefined;
   readonly _type: 'root' | 'project' | 'file' | 'describe';
+  readonly _testTypeHash: string;
```

Comment:
In `TestCase` we have `_testType: TestTypeImpl`. Perhaps we can do the same here and compare instances instead?